source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3794,10 +3831,12 @@ export class SubmitAction extends Action {
 }
 
 export class OpenUrlAction extends Action {
+    static readonly JsonTypeName = "Action.OpenUrl";
```

Comment:
> "Action.OpenUrl" [](start = 34, length = 17)

Nit: We should start considering adding a Constants module or class equivalent where these string literals can be housed instead of them being sprinkled all over the code (and potentially duplicated etc)

Reply:
Well, that's precisely what this is doing, and it's contextual to the action class, which is a better option for this particular case. I considered doing the same for element names in this PR but deferred to a later time.